Environment International 155 (2021) 106694

## Variations of antibiotic resistome in swine wastewater during full-scale anaerobic digestion treatment

### A B S T R A C T

Keywords:
Livestock waste
Anaerobic digestion
Antibiotic resistome
Microbial community
Pathogens

Anaerobic digesters have been widely used to treat wastewaters in livestock farms. With the increasing risk of antibiotic resistance originated from livestock husbandry, removal of antibiotics and antibiotic resistance genes (ARGs) via anaerobic digesters deserved more attention. Here we investigated the removal of antibiotics and ARGs in swine wastewater by three on-farm full-scale anaerobic digesters, including buried biogas digester (BBD), up-flow anaerobic sludge blanket (UASB) and high density polyethylene covered biogas digester (HDPE-BD). Variations of antibiotic resistome in swine wastewater were further revealed by metagenomic sequencing.

Results showed the removal efficiencies for antibiotics, ARGs and mobile genetic elements (MGEs) varied in the three digesters, ranging from 65.1% to 98.1%, 3.5%-71.0% and 26.9%-77.2%, respectively. In general, UASB and HDPE-BD showed better removal efficiencies than BBD. However, enrichment of metal resistance genes (MRGs) was noted in UASB. Pathogens could not be effectively removed by all of the three digesters. What’s more, accumulation of pathogens was found in UASB (removal efficiencies: 8.5%-13.6%). Bacterial community succession, horizontal genetic transfer and biocide and metal resistance genes (BMRGs) profiles jointly structured the variation of antibiotic resistome during anaerobic digestion. A total of 334 high-quality bins were identified from swine wastewater, 96 of which belonged to phylum of Firmicutes, Bacteroidetes and Proteobacteria carried ARGs. Proteobacteria was the dominant multi-drug resistant flora. Meanwhile, ARG-carrying pathogens (Bacteroides and Mycolicibacter) were found in the swine wastewater, suggesting a potential threat to human and animal health. The findings from this study showed that HDPE-BD is the most eco-friendly and effective anaerobic digester in controlling risks from antibiotic resistance determinants in swine wastewater.

## 1. Introduction

Antibiotics are widely used in livestock breeding for treating diseases and promoting growth. After application, parent compounds and metabolites could be released via urine and feces due to the low absorption and metabolism of antibiotics by animals. The antibiotic residues in swine wastes may cause the development and dissemination of antibiotic-resistant bacteria (ARB) and antibiotics resistance genes (ARGs) around the livestock breeding environment, and further pose potential risks to human health. Therefore, animal wastes need to be treated before discharge.

Anaerobic digestion is a robust biological treatment technology in the absence of oxygen to degrade organic matter while producing biogas via microbial metabolism. Due to low energy consumption, high biogas yield, straightforward operation and digestate nutrients for crops, anaerobic digestion has been widely used to treat livestock wastes. Types of anaerobic digesters mainly include buried biogas digester (BBD), up-flow anaerobic sludge blanket (UASB) and high density polyethylene covered biogas digester (HDPE-BD).

As a traditional waste treatment technology, anaerobic digestion has been evaluated against antibiotics and ARGs in some studies. Our previous study showed UASB and BBD could effectively remove antibiotics from swine wastewater, including sulfonamides, tetracyclines and lincomycin. Resende et al. constructed a lab-scale anaerobic digestion fed with dairy cattle manure, and found anaerobic digestion could effectively reduce the abundance of ermB, aphA2 and blaTEM-1. Ju et al. found lab-scale anaerobic sludge fermentation could remove 20%-52% of total ARGs by metagenomic sequencing. Nevertheless, significant quantities of antibiotics or ARGs can be detected in the digestate in some cases. Ma et al. found the abundance of ermB, ermF and tetW significantly increased during mesophilic anaerobic digestion. Wallace et al. sampled from a full-scale advanced anaerobic digester, and found the abundance of tetW increased in the digestate, compared with influent. Zhang et al. found both the lab-scale thermophilic and mesophilic anaerobic digestion could not reduce the abundance and diversity of total ARGs via metagenomic sequencing. In view of the inconsistence of ARGs removal during anaerobic digestion, researchers devote themselves to enhancing the removal of ARGs via the control of operation conditions in lab-scale anaerobic digestion, such as temperature, retention times and additives. However, most of the studies mainly focus on the lab-scale anaerobic digestion, little information has been known about the shared characteristics and differences of antibiotic resistome in different on-farm full-scale anaerobic digesters for actual animal wastewater treatment.

This study aimed to investigate the variations of antibiotics and antibiotic resistome in the process of full-scale swine wastewater anaerobic digestion with different technologies by using HPLC-MS/MS and metagenomic high-throughput sequencing, and explore the potential influencing mechanisms for the variations of antibiotic resistome by analyzing the associations among ARGs, bacterial community, mobile genetic elements, biocide and metal resistance genes. Finally, high-quality genomic drafts were assembled via binning, and annotated to both taxonomy and ARGs to reveal the ARG-carrying hosts during anaerobic digestion. The results will facilitate better understanding of antibiotic resistome and related risks during anaerobic digestion of swine wastewater.

## 2. Materials and methods

### 2.1. Full-scale anaerobic digesters

Three full-scale anaerobic digesters selected for this study were equipped for the wastewater treatment in three swine farms (LC, ST and DC) located in Yunfu of Guangdong Province, South China. The technologies of anaerobic digestion units in the farms LC, ST and DC were BBD, UASB and HDPE-BD, respectively, with the technical details and farm information being given in Table S1. The wastewater of primary settling tanks from the three farms was regarded as the influent of anaerobic digesters, which was collected along with the effluent of anaerobic digesters on Jun 8, 2016. 1 L and 2.5 L of time integrated composite wastewater samples were collected respectively for chemical and biological analysis by using amber glass bottles and sterile plastic bottles. Meanwhile, the temperature (T), pH, dissolved oxygen (DO), redox potential (RP) and electrical conductivity (EC) of the sampled wastewater were recorded by using a YSI-Pro2030 multi-parameter water quality monitor (YSI Incorporated, USA) (Table S2). The samples for chemical analysis were added with 4 M H2SO4 to adjust pH to 3, and 5% methanol to suppress the microbial activities. Then the collected samples were transported into laboratory in a cooler and then stored in a cold room (4◦C).

### 2.2. Antibiotics extraction and instrumental analysis

41 target antibiotics, including 13 sulfonamides, 5 tetracyclines, 12 fluoroquinolones, 6 macrolides and 5 other types of antibiotics were extracted from the wastewater by using solid phase extraction (SPE) method. The detailed information of the target antibiotics is listed in Table S3. The instrumental analysis was carried out by using the Agilent 1200 series ultrahigh-performance liquid chromatograph coupled to an Agilent 6460 triple quadrupole mass spectrometer with electrospray ionization under positive ionization mode (UHPLC-ESI-MS-MS). The instrumental conditions, limits of detection (LOD), limits of quantification (LOQ), and methods of statistical analysis can refer to our previous study.

### 2.3. DNA extraction and real-time qPCR

Total genomic DNA was extracted from the filters of wastewater samples in triplicate according to the manufacturer’s protocol of Powersoil DNA Isolation Kit. The quality of the extracted DNA was checked via 1% gel, NanoDrop 2000 and Qubit 3.0. Then the 16S rRNA, E. coli biomarker gene (uida) and clinically key antibiotic resistance gene (mcr-1 and bla TEM) were quantified by using real-time qPCR based on the ViiA 7 Real-Time PCR System. The primers, annealing temperature and expected amplicon sizes for those genes are listed in Table S4.

### 2.4. Metagenomic sequencing

Genomic DNA extracts of the 18 wastewater samples were selected and sent to Magigene for metagenomic sequencing. 150 bp paired-end metagenomic high-throughput sequencing was performed on the Illumina Hiseq X Ten platform. Total 221 Gbp of raw data was collected from the 18 wastewater samples (Table S5). After filtered by using Trimmomatic software, 10.7-Gbp clean data of each wastewater sample was obtained to yield a total of 193 Gbp. The trimmed metagenome reads were assembled to scaffolds by MEGAHIT (Version 1.0). The metagenomic raw data were deposited into the National Center for Biotechnology Information Short Reads Archive database.

### 2.5. Bioinformatic analysis

The methods for the bioinformatics analysis in this study can be referred to our previous study. In brief, bacterial species annotation was performed based on blast results against NCBI-NR database via Diamond software and MEGAN 4 software. The pathogenic annotation was carried out based on the Pathogen Host Interaction database. The identification of ARGs was conducted by using BLASTX against SARG database with e-value ≤ 1e-5 via ARGs-OAP online. The clean data was blasted for MGEs (Integrons, gene cassettes, insert sequences, transposons and plasmids) annotation against an offline database referencing to the previous studies. With growing concern about the biocide and metal resistance for bacteria, the biocide and metal resistance genes (BMRGs) were searched against BacMet database using BLASTX with e-value ≤ 1e-5. 2.6. Binning analysis
Whole genome de novo assembles were performed via MEGAHIT (http://github.com/voutcn/megahit, parameters: k-min 35, k-max 115, k-step 20) based on the clean data from all of the influents and effluent, respectively. Binning of assembled metagenomic sequences was executed by using MetaBAT2 with default parameters-“sensitive” based on read coverage, coverage variance and tetranucleotide frequencies. The completeness and contamination of bins were valued using CheckM with default parameters. In order to reveal the ARG-carrying hosts in influent and effluent from anaerobic digesters, high quality genome bins (completeness ≥ 70%, contamination ≤ 4%) were chosen for taxonomy and ARGs annotation via GTDB-Tk software and blasted against GTDB database with default parameters and SRAG database (e-value ≤ 1e-7, similar identity ≥ 60%, alignment length ≥ 75 bp). Meanwhile, the ARG-carrying bins were conducted with integrase and insert sequence annotation to reveal the potential of ARG horizontal transfer (e-value ≤ 1e-7, similar identity ≥ 60%, alignment length ≥ 50 bp).

2.7. Data analysis
Potential resistance selection risks for bacterial community posed by the antibiotics in the swine wastewater were calculated via the risk quotient (RQ) approach. The RQRS values of antibiotics were assessed by the following formula:

RQRS = MEC/PNECRS.

Where MEC represents the measured environmental concentration in wastewater; PNECRS means the “predicted no-effect concentration for antibiotic resistance selection” (Table S6). The RQ values were divided into three levels of risk: 0.01–0.1, low; 0.1–1, medium; and above 1, high risk. Variance analysis was performed by using SPSS 13.0 software via one-way ANOVA analysis. “ppm” was used as the unit of the relative abundance of pathogens, ARGs, MGEs and BMRGs, and one ppm represents one read in one million reads. Principal

| Samples          | OTC | DC | EFX | CTC | LIN | SMM | OFX | NFX | SDZ | BACI | TC | ETM-H2O | CFX | TMP |
|------------------|-----|----|-----|-----|-----|-----|-----|-----|-----|------|----|---------|-----|-----|
| BBD_In           |     |    |     |     |     |     |     |     |     |      |    |         |     |     |
| BBD_Ef           |     |    |     |     |     |     |     |     |     |      |    |         |     |     |
| UASB             |     |    |     |     |     |     |     |     |     |      |    |         |     |     |
| HDPE-BD_In       |     |    |     |     |     |     |     |     |     |      |    |         |     |     |
| HDPE-BD_Ef       |     |    |     |     |     |     |     |     |     |      |    |         |     |     |

Concentrations (a) and potential resistance selection risks (b) of antibiotics in the influent and effluent of the three anaerobic digesters (BBD, UASB and HDPE-BD). In indicates influent, while Ef indicates effluent. OTC, oxytetracycline; DC, doxycycline; EFX, enrofloxacin; CTC, chlorotetracycline; LIN, lincomycin; SMM, sulfamonomethoxine; OFX, ofloxacin; NFX, norfloxacin; SDZ, sulfadiazine; BACI, erythromycin-HO; bacitracin; TC, tetracycline; CFX, ciprofloxacin; ETM-H2O, TMP, trimethoprim. Component Analysis (PCA) was performed based on the abundance of the antibiotic resistome, bacterial community, MGEs and BMRGs profile during the anaerobic digestion of swine wastewater via Canoco 5.0 and R3.5.1. Variation partitioning analysis (VPA) and procrustes analysis were conducted to reveal the relationships among the wastewater samples from anaerobic digesters.

| Phylum                  |   |
|------------------------|---|
| Arcobacter              | 2 |
| Lactobacillus          | 7 |
| Others                 | L |
| Desulfovibrio         |   |
| Fusabacteria           |   |
| Delnococcus Themus     |   |
| Corynebacterium        |   |
| Chlorobi               |   |
| Acinotobactor          |   |
| Vorucomlcrobla        |   |
| Sphaerochaeta         |   |
| Syncrgistctcs         |   |
| Nitrospino             |   |
| Prevotella             |   |
| Spirochaetes           |   |
| Peehvlororciga        |   |
| Buctcroidetus         |   |
| Peptostreptococcaceae_noname |   |
| Flmmlcune $           |   |
| Alicycliphilus        |   |
| Proteobacteria        |   |
| Actinobactena         |   |
| Seucobiacteulum       |   |
| Dermatophilaceae_unclassified |   |
| Sulfurospirillum      |   |
| Escherichla           |   |
| Comamonas             |   |
| Nitrosospira          |   |
| Pasteurella           |   |
| Thauera               |   |
| Aerococcus            |   |
| Nitrobacter           |   |
| Bacteroides           |   |
| Bifidobacterium       |   |
| Hydrogenophilaceae_unclassifiod |   |
| Erysipelotrichaceae noname |   |
| Thiomonas             |   |
| Thiobacillus          |   |
| Hyphomicroblum       |   |
| Acholeplasma          |   |
| Acidithiobacillus     |   |
| Allochromatium        |   |
| Rhodopseudomonas      |   |
| Brachybacterium       |   |
| Streptococcus         |   |
| Clostridium           |   |
| Parabacleroides       |   |
| Halothiobacillus      |   |
| Pligudoxanthomonas    |   |
| Brevundimonas         |   |
| Succinispira          |   |
| Brevibacterium        |   |
| Dysgonomonas          |   |
| Snhigodracobaaterium  |   |
| Brachymonas           |   |
| Dictzia               |   |

| Variance               |   |
|------------------------|---|
| PC1 (33% variance)     |   |
| PC2 (16%)              |   |

| Phylums:               |   |
|------------------------|---|
| Dermatophilaceae_unclassified |   |
| Peptostreptococcacede |   |
| Escherichia           |   |
| Desulfovibrio         |   |
| Hyphomicrobium       |   |
| Firmicutes            |   |
| Actinobacteria        |   |
| Proteobacteria        |   |
| Bacteroidetes         |   |
| Spirochaetes          |   |

| Water samples          | Relative abundance (log10 value) |
|------------------------|----------------------------------|
| connedagteriun         | 1.0                              |
| Lactobacillus          | 412                              |
| Prevotella             | 41.4                             |
| Hydrogenophllaceae     | 4.6                              |
| Methylosarcina         | -1.8                             |
| Acinetobacter          | 2.0                              |
| Acidithiobacillus      |   |
| Thauera                |   |
| Comamonas              |   |
| Sphaerochaeta         |   |
| Sulfurospirillum       |   |
| Arcobactcr            |   |

Fig. 2. Microbial community of influent and effluent from the three anaerobic digesters. (a), The variation of relative abundance of bacteria at phylum level; (b), PCA of the bacteria at genus level; (c), The heatmap of relative abundance of dominant bacteria (top 50); and (d), Shared and unique biomarker genera among swine wastewater according to LefSe analysis (LDA score >4). were obtained and revised based on LefSe analysis and their relative abundance, and then the visualization of unique and shared biomarkers of genera and ARG subtypes was carried out via Cytoscape 3.7.1 software.

## Results and discussion

### Occurrence and removal of antibiotics

Total 16 of 41 target antibiotics were detected in the influent and effluent of anaerobic digesters from the swine farms, including 3 sulfonamides, 4 tetracyclines, 4 fluoroquinolones, 2 macrolides, lincomycin, trimethoprim and bacitracin. Oxytetracycline (OTC) was the most predominant target compound in all the wastewater samples, with the concentrations up to 2,040,000 ± 390,000 ng/L, followed by sulfachlorpyridazine (SCP, up to 27,300 ± 2,800 ng/L) and sulfamonomethoxine (SMM, up to 16,100 ± 304 ng/L). The prevalence of OTC, SCP and SMM in the swine wastewater could be attributed to its extensive usage in feed and low absorption by animal gut. The total antibiotic concentration of the influent from HDPE-BD (2,070,000 ± 394,000 ng/L) was significantly higher than that from BBD (191,000 ± 52,400 ng/L) and UASB (310,000 ± 80,400 ng/L) (p < 0.01). This could be attributed to the different feeding strategies between fattening pig (DC) and sow (LC and ST). The antibiotic concentrations in the effluents ranged from 39,900 ± 1,700 ng/L to 66,700 ± 9,840 ng/L. The removal of detected antibiotics by the anaerobic digestion from the BBD, UASB and HDPE-BD were 65.1%, 82.6% and 98.1%, respectively. As the most dominant antibiotic, the removal of OTC from wastewater mainly relied on the intense sorption of granular matter because of strong interaction with clay, organic matter and metal oxides. According to the risk quotient (RQ) approach, OTC, enrofloxacin (EFX) and chlorotetracycline (CTC) in the swine wastewater posed high resistance selection risks for bacterial community, and which showed a declined trend after anaerobic digestion.

### Changes of bacterial community and pathogens

According to Figure S1, the absolute abundance of 16S rRNA gene in the influent was significantly higher than the effluent (p < 0.05), indicating decline of bacterial abundance after anaerobic digestion. The bacterial species annotation results showed that 12 of dominant phyla were detected in all the swine wastewater samples. The dominant phyla in the influent included Proteobacteria, Actinobacteria, Firmicutes and Bacteroidetes, which accounted for more than 67.7% of the bacterial community. This is similar to the bacterial composition of swine manure in the previous studies. After anaerobic digestion, Proteobacteria became the absolutely dominant flora in the effluents. Similar phenomenon was reported by Guo et al. Proteobacteria is the top contributor to the anaerobic fermentation process, such as conversion of glucose, butyrate, propionate and acetate except for methanogenesis during anaerobic digestion. However, Firmicutes was dominated in the most full-scale anaerobic digestion systems fed with cattle manure and swine manure. Generally, stable anaerobic digestion systems with enough nutrients often show a dominance of Firmicutes over Proteobacteria. The predominance of Proteobacteria in the effluents from the three anaerobic digesters may result from the consumption of nutrients in favor of the Firmicutes in the core area of anaerobic fermentation due to the sufficient hydraulic retention time and lack of stirring. Noteworthily, Spirochaetes accumulated in the effluent of BBD and HDPE-BD. As a kind of Gram-negative heterotrophic bacteria, Spirochaetes prefer weakly basic and anaerobic conditions, and have been frequently found in anaerobic digesters. The weakly basic and relatively low dissolved oxygen (DO) condition in the effluent of BBD and HDPE-BD led to the accumulation of this phylum in comparison with UASB. According to the PCA analysis, the bacterial community structure between the influent and effluent had more obvious difference for UASB and HDPE-BD than that for BBD. The dramatic variations of DO, oxidation redox potential (ORP) and electrical conductivity (EC) between the influents and effluents from UASB and HDPE-BD could explain their difference of bacterial community versus BBD. At genus level, total 164 genera were identified in the swine wastewater samples, and the dominant genera included Arcobacter, Lactobacillus, Desulfovibrio, Corynebacterium, Peptostreptococcaceae_noname, Acinetobacter, Sphaerochaeta, and Pseudomonas. LefSe analysis was performed to identify the biomarker genera belonging to the influents and effluents, respectively. Thereafter, the unique and shared biomarker genera (LDA score >4) were visualized via network by using Cytoscape software. The shared biomarker genera with high relative abundance among the influents from the three anaerobic digesters included Lactobacillus, Acinetobacter, Corynebacterium and Prevotella, which were frequently identified in the untreated swine waste and reduced after anaerobic digestion. Lactobacillus sobrius even could be considered as the unique biomarker of swine manure contamination. However, the shared biomarker genera among the three effluents was found much less than the influents. Sphaerochaeta belonged to Spirochaetes phylum was the only genus shared between the effluents from BBD and HDPE-BD. The unique biomarker genera from the UASB effluent were abundant and mainly belonged to Proteobacteria. Thereinto, Methylosarcina obviously accumulated in the effluent of UASB. Methylosarcina is an important methanotroph, and prefers to metabolize dissolved methane in UASB. The unique biomarker genera from the BBD and HDPE-BD effluents were also few, mainly Arcobacter and Sulfurospirillum, respectively. The prevalence of Arcobacter in the effluents severely threaten the health of human and swine, which can cause multiple zoonotic diseases, including abortion, diarrhea, bacteremia, endocarditis and peritonitis. Among the 14 types of pathogens at the first level from the PHI database, “Reduced virulence”, “Unaffected pathogenicity”, “Effector (plant avirulence determinant)” and “Loss of pathogenicity” were dominant. Though anaerobic digestion could significantly reduce the absolute abundance of Escherichia coli, the relative abundance of pathogens in the effluents just slightly declined (11.0%-13.6% removal) or even increased in UASB effluent (-8.5% removal). This result indicated anaerobic digesters could not thoroughly eliminate the risk of pathogens. At the species level, total 205 pathogenic species were identified among the swine wastewater samples. The dominant pathogens included Salmonella enterica, Fusarium graminearum, Magnaporthe oryzae, Pseudomonas savastanoi and Asperigillus fumigatus. It was notable that some hazardous and lethal pathogens such as Vibrio cholera, Yersinia pestis, Neisseria meningitides, Brucella abortus and Brucella melitensis were found in the swine wastewater. Their occurrence in the digestate might pose severe healthy risks to people and livestock animals. For instance, more than 56 public health emergencies of brucellosis (Brucella abortus and Brucella melitensis) broke out in 2019, according to the annual global brucellosis surveillance report. Meanwhile, the enrichment of crop pathogens (Magnaporthe oryzae and Fusarium graminearum) may hinder the direct application of digestate.

### Variations of ARGs

According to the ARGs annotation, total 21 ARG types were detected in the influents and effluents from the three anaerobic digesters. The predominant ARG types included tetracycline, aminoglycoside, macrolide-lincosamide-streptogramin (MLS), and | Antibiotic            | Removal Efficiency |
|----------------------|--------------------|
| Tetracenomycin C     | 3.5% removal        |
| Puromycin            | 71.0% removal       |
| Carbomycin           | 62.2% removal       |
| Fosfomycin           |                    |
| Quinolone            |                    |
| Kasugamycin          |                    |
| Rifamycin            |                    |
| Polymyxin            |                    |
| Fascariocircin      |                    |
| Bleomycin            |                    |
| Trimethoprim         |                    |
| Unclassified          |                    |
| Bacitracin           |                    |
| Beta-lactam         |                    |
| Chloramphenicol      |                    |
| Sulfonamide          |                    |
| MLS                  |                    |
| Aminoglycoside       |                    |
| Tetracycline         |                    |

The prevalence of those ARGs in swine wastewater could be ascribed to the wide use of those antibiotics in swine breeding and disease treatment. The total relative abundance of ARGs in the influents of UASB and HDPE-BD (2,660 ppm and 2,190 ppm) were higher than that from BBD (935 ppm). The difference may result from the different farm operations and pre-treatment process for swine wastes. The wastewater from farm LC was filtrated by a screen after solid–liquid separation, and flowed into the BBD, while the wastewater from farms ST and DC was simply solid–liquid separated. The low concentration of antibiotics in the BBD influent posed a lower selection pressure on bacterial community when compared with USAB and HDPE-BD. The removal efficiencies for the relative abundance of ARGs in BBD, UASB and HDPE-BD were 3.53%, 71.02% and 62.19%, respectively. It indicated more effective removal by the UASB and HDPE-BD in comparison with the BBD. In addition, the absolute abundance of the key clinically important antibiotic-resistant genes mcr1 and blaTEM reduced after anaerobic digestion by 94.0%-100% and 91.1%-97.9% respectively. Among all the swine wastewater samples, total 495 ARG subtypes were identified, and the number of ARG subtypes (ranging between 305 and 436) detected in all the influents was higher than that in all the effluents (268–298). The PCA result showed that the ARG profiles between the influents and effluents from UASB and HDPE-BD showed more obvious differences than that from BBD, which was similar to the bacterial community. The reduction in relative abundance and diversity of ARGs during anaerobic digestion could be ascribed to the weak competitive ability of ARG hosts due to fitness costs with the remittence of antibiotic selection pressure. The antibiotic resistome in anaerobic digesters was found to link with the succession of microbial community. So the slight decline of relative ARG abundance for BBD may be explained by the small variations of bacterial community during anaerobic digestion.

The dominant ARG subtypes among the swine wastewater samples included aadA, aadE, aph(3′’’)-I, ermF, mefA, sul1, sul2, tetM, tet36, tetQ, tet44, and tetW. The prevalence of tetracycline and sulfonamide resistance genes could be explained by the intense selection pressure of OTC, SMM and SCP. LefSe analysis was also used to reveal the ARG biomarkers between influent and effluent for each anaerobic digester, and then the shared and unique biomarkers among the three anaerobic digesters were visualized in the network system. Due to the little difference of ARG profiles between the influent and effluent of BBD, their respective ARG biomarkers were less in comparison with. | ARG Types         | Relative Abundance (Log10 value) |
|-------------------|----------------------------------|
| Sulfonamide       | 2.2                              |
| Multidrug         | 2.0                              |
| Tetracycline      | 1.8                              |
| Aminoglycoside    | 1.6                              |
| Fosmidomycin      | 1.4                              |
| Chloramphenicol   | 1.2                              |
| Beta-lactam      | 1.0                              |

Fig. 4. Shared and unique biomarker ARG subtypes among the influents and effluents from the three anaerobic digesters.

The shared ARG biomarkers among the influents of UASB and HDPE-BD mainly included tetQ, tetW, aph(3′’’)-III, ermF and chloramphenicol exporter gene, which indicated the effective removal during anaerobic digestion. The unique ARG biomarkers from the UASB and HDPE-BD influents included tetL, ermB and lnuB, and sul2, floR and tetX, respectively. As for the effluents, the shared ARG biomarkers included tet44 and tetC. Meanwhile UASB effluent had the most ARG biomarkers among the three effluent, including tetG, mdtB and mdtC. These ARGs enriched significantly after anaerobic digestion deserved more attention. ### 3.4. Variations of MGEs and BMRGs

According to the MGEs annotation, total 378 integrons & gene cassettes, 1,234 insert sequences & transposons and 2,432 plasmids were identified among the swine wastewater samples. The total relative abundance of MGEs in the influents of BBD, UASB and HDPE-BD were 1,315 ppm, 6,075 ppm and 3,946 ppm respectively. Similar to the ARGs, UASB and HDPE-BD could effectively reduce 70.3% and 77.2% of the relative abundance for MGEs, which was helpful to reduce the potential of ARG horizontal gene transfer. The reduction of MGEs could be ascribed to the decline of MGE host due to metabolic burden and plasmid instability. The dominant insert sequences & transposons, integrons & gene cassettes and plasmids included TnAs3, ISBbi1, ISCco2, tnpA_c, intI1_i, sul3_c, NZ_CP018072.1, NZ_CP019443.1 and NZ_CP020359.1. In general, intI1 was frequently described to be closely linked with ARGs, and played a key role in horizontal genetic transfer among bacteria. But other dominant MGEs and their role in ARGs transfer process in the environment should also be concerned. For instance, TnAs3 was associated with rapid dissemination of blaOXA232 which confers carbapenem resistance to Klebsiella pneumonia. ISBbi1 was found located in the genome of an extensive resistant Chryseobacterium indologenes with 90 antibiotic resistance genes.

As for the biocide and metal resistance genes (BMRGs), 224 BRGs, 253 MRGs and 56 BRGs&MRGs were detected against BacMet database via metagenomic sequencing. The total relative abundances of BMRGs in the influents from BBD, UASB and HDPE-BD were 0.0839 ppm, 0.104 ppm and 0.102 ppm. The predominant MRGs, MRGs & BRGs and BRGs included modC, recG, znuC/yebM, baeR, baeS, merA, and evgS. Zinc (Zn), Nickel (Ni) and Copper (Cu) resistance genes were the primary MRG types in swine wastewater, similar to the previously reported study. The prevalence of these MRGs in swine wastewater could be explained by the wide use of Cu, Zn and Ni in commercial feeds and additives. The BBD and HDPE-BD could significantly reduce the relative abundance of MRGs (p < 0.01), while MRGs accumulated in the effluent of UASB compared with its influent (p < 0.01). This might be attributed to the metal materials of UASB container. It was found that corroded iron pipe could release Fe, Mn, Ni, Al, Zn and Cu into water under stagnant and active flow conditions, with the concentration up to 50 μg/L. Hence the release of metals from UASB container may increase heavy metal selection pressure on bacteria, eventually resulting in the enrichment of MRGs.

### 3.5. Effect of bacterial community, MGEs and BMRGs on the antibiotic resistome

Procrustes analysis was used to assess the overall correlations for the antibiotic resistome with bacterial community, MGE profiles and BMRGs profiles. Bacterial community was significantly relative to the antibiotic resistome (M2 = 0.5584, p = 0.02), which indicated bacterial phylogeny structured the variation of antibiotic resistome during anaerobic digestion of swine wastewater. Based on procrustes analysis, some published studies figured out microbial phylogeny may be the dominant influencing factor for antibiotic resistome in the environment, such as landfill leachate, groundwater near a municipal solid waste landfill, and sewage treatment plant. MGE profiles also had significant correlations with antibiotic resistome (M2 = 0.3525, p = 0.001) during anaerobic digestion. This result indicated horizontal genetic transfer mediated by MGEs was a crucial factor driving the variation of antibiotic resistome during anaerobic digestion. Conversely, MGE profiles were not the key determinants for the variations of antibiotic resistome during manure composting (procrustes analysis, M2 = 0.912, p = 0.303) in a previous study. The difference may be ascribed to the inhibition of transfer ability for bacteria due to the low moisture during composting, in comparison with the high mobility in aquatic environment during anaerobic digestion. The use of biocides and metals in swine feeding could also pose the co-selective pressure for antibiotic resistance. BMRG profiles also showed significant association with antibiotic resistome during anaerobic digestion (M2 = 0.1308, p = 0.001). Meanwhile according to the VPA, the dominant phyla, MGEs and BMRGs could explain 99.9% of the variations of ARGs profile during anaerobic digestion. Thereinto, up to 56.3% of variations for antibiotic resistome was attributed to the joint effects of these three factors. In a word, bacterial community succession, horizontal genetic transfer and BMRG profiles jointly structured the antibiotic resistome during anaerobic digestion.

### 3.6. Identification of ARG-carrying hosts via binning analysis

Binning analysis was used to reveal the ARG-carrying hosts in the influents and effluents of swine wastewater anaerobic digestors. 162 and 172 of high-quality bins (completeness ≥ 70%, contamination ≤ 4%) were identified from the whole metagenomes of the composite influents and effluents, respectively. Due to the common limitation of completeness of bins and reference sequences for genomic taxonomy annotation, only 57.8% and 8.1% of bins were annotated at genus and species level. Thereinto, many bins were annotated with non-classical names or rarely reported, such as genus_UBA1206, genus_CAG-1024 and Paenalcaligenes, which expanded the knowledge of the diversity of bacteria in swine wastewater. According to blasting results against SARG database, 38 and 38 bins were identified as ARG-carrying host in the influents and effluents, respectively. Those ARG-carrying bins from the influents were annotated to 6 phyla carrying 23 different ARG subtypes, and the majority of bins belonged to Firmicutes and Bacteroidetes. Meanwhile, ARG-carrying bins from the effluents were classified into 10 phyla carrying 27 different ARG subtypes. Thereinto, most of ARG hosts were identified to Proteobacteria, which was in accordance with its predominance in the effluents of anaerobic digesters. The predominance of ARG hosts from Firmicutes, Bacteroidetes and Proteobacteria was in line with the previous studies. Meanwhile, a plenty of integrase genes and insert sequences were identified in those ARG-carrying bins. Thereinto, intI1 and ISCte3 were located in the adjacent area of ARGs, including acrB, multidrug transporter, oprM, OXA-9 and rosA, which verified their potential of horizontal gene transfer during swine wastewater treatment.

Multi-drug resistant bacteria have been a severe threat for public health, bins carrying four or more ARG subtypes as representative multi-drug resistant host. In the present study, total 30 ARG-carrying genomes carried more than 2 types of ARG subtypes. Thereinto, 6 bins were the representative multi-drug resistant hosts, including 4 bins belonged to Proteobacteria and 1 bin belonged to Desulfobaterota. The bin belonged to family_Spongiibacteraceae carried up to 7 ARG subtypes, including rosA, macB, mdtB, mexF, mexT, arnA and RP(cAMP). The characteristic of multi-drug resistance for Proteobacteria agreed with the results from previous studies, in which up to 65.45% of representative multi-drug resistant genomes were annotated to Proteobacteria. It indicated Proteobacteria may be the dominant multi-drug resistant microbes during the full-scale anaerobic digestion of swine wastewater. Meanwhile, the occurrence of antibiotic-resistant Bacteroides and Mycobacterium carrying bacA and aac(2′)-I was noteworthy, as some pathogens from the two genera could cause lethal animal diseases, i.e. septicemia, leprosy and tuberculosis. | Holo                            | Acidobacteria         | Copy number of ARGs |
|---------------------------------|----------------------|----------------------|
| in.bin148                       |                      |                      |
| eff bin195                      |                      |                      |
| in.bin112                       | Eggerthella          |                      |
| effbin131                       | 20cm-4-69-           | Actinobacteria       |
| eff bin80                       | Mycolicibacter       |                      |
| in.bin126                       | Proteiniphilum       |                      |
| in.bin128bin129                 | Petrimonas           |                      |
| in.bin146                       |                      |                      |
| in.bin160                       | Progobocteraceceae   | Proteiniphilum       |
| bin350                           |                      |                      |
| bin47                            |                      |                      |
| in.bin86                        | Bacteroides          | Bacteroidetes        |
| effbin11~                       | ELB16-189            | aminoglycoside       |
| effbin121                       | PHOS-HE28            |                      |
| effbin136                       | Palsa-955            | bacitracin           |
| effbin162                       | UBA1952              | beta lactam         |
| effbin42                        |                      |                      |
| embina8                         | NS-102               |                      |
|                                 | Palsa                |                      |
|                                 | Falsaie55_           | chloramphenicol      |
|                                 | fosmidomycin         |                      |
| effbin150                       | OLB10                | kasugamycin          |
| effbin338                       | UBA6126              | Caldisericota       |
| eff bin280                      | Sulfurospirillum     | Campylobacterota     |
|                                 | multidrug            |                      |
| bin100                           | Desulfobulbus        |                      |
| bin123                           | 9_DesuifomicrobiaceaeUBA1062 | rifamycin |
| bin208                           |                      |                      |
| bin220                           |                      |                      |
|                                 | Desulfobacterota     |                      |
| bin284                           | Desulfovibrio       |                      |
|                                 | Desulfovibrio       | unclassified         |
| bin108                           | UBA4871              | vancomycin           |
| bin116                           | MS_                  |                      |
| bin139                           |                      |                      |
|                                 | ceac                 |                      |
| hint                             |                      |                      |
| bin149                           |                      |                      |
| bin188                           | 583288bhomona        |                      |
| bin200                           | Fastidiosipilaccae   |                      |
|                                 | Helcococcaceae       |                      |
| bin247                           | 9 Rueoinococ        |                      |
| bin258                           | Ruminococcaceae      |                      |
| in.bin260bin273                 | DTU036               |                      |
| bin280                           | 9 BucocicoccaceaeLactobacillu | Firmicutes |
| in.bin2gg                       |                      |                      |
| in.bin300                       | Helcococcaceae       |                      |
| in.bin334bin327                 | UBA2212              |                      |
|                                 | Ndongobacter         |                      |
|                                 | UBA636               |                      |
| bin353                           |                      |                      |
| bin363                           | Streptococcus        |                      |
| in bin67                        | Butyrivibrio        |                      |
| effbin182                       | UBA1447              |                      |
| effbin184                       | {sacchococecrmentanaceaeTrichococcus | |
| eff bin202                       |                      |                      |
| eff bin237                       | CAG-1024             |                      |
| effbin342                       | Christensenellales    |                      |
| effbin125                       | PolyangiaceaeUBA6777 | Myxococcota         |
| effbin49                        |                      |                      |
| effbin47                        | Thermoguttaccac      |                      |
|                                 | Ehiopseatoeronas     | Planctomycetota     |
| bin205                           |                      |                      |
| bin226                           |                      |                      |
| in.bin269                       | Succinivibrio       |                      |
| in.bin328                       |                      |                      |
| eff:bin106                      |                      |                      |
| eff:bin160                      | Thiobacillus         |                      |
| effbin1                         | Alcanivoracaceae     |                      |
| effbin165                       | Ga0077527            |                      |
| eff bin189                      | Nitrosomonas         | Proteobacteria       |
| effbin198                       | GCA-2729495          |                      |
| effbin236                       | 21-64-14             |                      |
| eff bin239                      | Nitrosomonas         |                      |
| effbin246                       | Gammaproteobacteria   |                      |
| effbin268                       | UBA7239              |                      |
| eff bin401                      | UBA6156              |                      |
| effbin82                        | Steroidobacteraceae   |                      |
| effbin89                        | Spongiibacteraceae    |                      |
| effbingo                        | Methylosarcina       |                      |
| effbin166                       | Kiritimatiellae      |                      |
| effbin213                       |                      |                      |
| effbin66                        | Kiritimatiellae      | Verrucomicrobiota    |
|                                 | Prosthecobacter       |                      |

Fig. 5. Distribution and copy numbers of ARGs in the bins with high quality (completeness ≥ 70%, contamination ≤ 4%) from influent and effluent of anaerobic digestion. c: class; o: order; f: family; g: genus.

tuberculosis). Among the annotated 37 ARG subtypes, bacA was identified with the highest detection frequency (30.8%) throughout the 78 ARG-carrying genomic bins, which conferred bacterial resistance to bacitracin via inhibiting the dephosphorylation of undercaprenyl pyrophosphate necessary for the recycling of the lipid carrier, thereby ultimately hampering the biosynthesis of cell wall polymer. The occurrence of bacA even accounted for up to 50.1% of the presentative multi-drug ARGs-carrying hosts from activated sludge by Liu et al. The prevalence of bacA in swine wastewater could be ascribed to the extensive usage of bacitracin zinc in feed, which could be further confirmed by the detection of bacitracin in the present study. Therefore, the ubiquity of bacitracin resistance among pathogens impelled us to exploit its new replacement.

4. Conclusions

This study demonstrated the on-farm full-scale UASB and HDPE-BD could effectively remove antibiotics, ARGs and MGEs from swine | Gene                | 5000 | 5500 | 6000 | 6500 | 7000 | 7200 | 7400 | 7600 | 7800 | 16000 | 16400 | 16800 | 17200 | 34000 | 34200 | 34400 | 34600 | 34800 | 46000 | 46400 | 46800 | 47200 |
|---------------------|------|------|------|------|------|------|------|------|------|-------|-------|-------|-------|-------|-------|-------|-------|-------|-------|-------|-------|
| acrB                |      |      |      |      |      |      |      |      |      |       |       |       |       |       |       |       |       |       |       |       |       |
| intl1               |      |      |      |      |      |      |      |      |      |       |       |       |       |       |       |       |       |       |       |       |       |
| oprM                |      |      |      |      |      |      |      |      |      |       |       |       |       |       |       |       |       |       |       |       |       |
| OXA-9               |      |      |      |      |      |      |      |      |      |       |       |       |       |       |       |       |       |       |       |       |       |
| rosA                |      |      |      |      |      |      |      |      |      |       |       |       |       |       |       |       |       |       |       |       |       |

Gene arrangement of high-quality bins sequences together with ARGs and integrons or insert sequences

wastewater versus the full-scale BBD. More dramatic variations of bacterial community were observed between the influents and effluents from the UASB and HDPE-BD than the BBD. However, all the three anaerobic digesters could not effectively eliminate the pathogenic risk from the swine wastewater. The changes of bacterial community, horizontal genetic transfer and BMRG profiles jointly structured the antibiotic resistome during anaerobic digestion. Meanwhile, 76 out of 334 genomic bins with high-quality were annotated as ARG-carrying hosts, mainly belonged to Firmicutes, Bacteroidetes and Proteobacteria. As the dominant flora in the effluents, Proteobacteria was also the dominant multi-drug resistant flora. Additionally, the occurrence of ARG-carrying pathogens such as Bacteroides and Mycolicibacter should be paid more attention before application of digestate. M. Zhang et al.
Environment International 155 (2021) 106694

| Author(s) | Title | Journal | Year | Volume | Page |
|-----------|-------|---------|------|--------|------|
| Gurmessa, B., Pedretti, E.F., Cocco, S., Cardelli, V., Corti, G. | Manure anaerobic digestion effects and the role of pre- and post-treatments on veterinary antibiotics and antibiotic resistance genes removal efficiency | Sci. Total Environ. | 2020 | 721 | 137532 |
| He, L.Y., Ying, G.G., Liu, Y.S., Su, H.C., Chen, J., Liu, S.S., Zhao, J.L. | Discharge of swine wastes risks water quality and food safety: Antibiotics and antibiotic resistance genes from swine sources to the receiving environments | Environ. Int. | 2016 | 92–93 | 210–219 |
| Huson, D.H., Auch, A.F., Qi, J., Schuster, S.C. | MEGAN analysis of metagenomic data | Genome Res. | 2007 | 17 | 377–386 |
| Jia, S.Y., Zhang, X.X., Miao, Y., Zhao, Y.T., Ye, L., Li, B., Zhang, T. | Fate of antibiotic resistance genes and their associations with bacterial community in livestock breeding wastewater and its receiving river water | Water Res. | 2017 | 124 | 259–268 |
| Jiang, H., O’Callaghan, D., Ding, J.B. | Brucellosis in China: history, progress and challenge | Infect. Dis. Poverty | 2020 | 9 | 55 |
| Ju, F., Li, B., Ma, L.P., Wang, Y.B., Huang, D.P., Zhang, T. | Antibiotic resistance genes and human bacterial pathogens: Co-occurrence, removal, and enrichment in municipal sewage sludge digesters | Water Res. | 2016 | 91 | 1–10 |
| Kang, D.W.D., Li, F., Kirton, E., Thomas, A., Egan, R., An, H., Wang, Z. | MetaBAT 2: an adaptive binning algorithm for robust and efficient genome reconstruction from metagenome assemblies | Peerj | 2019 | 7 | e7359 |
| Kinyua, M.N., Rowse, L.E., Ergas, S.J. | Review of small-scale tubular anaerobic digesters treating livestock waste in the developing world | Renew. Sust. Energy Rev. | 2016 | 58 | 896–910 |
| Kong, W., Li, C., Dolhi, J.M., Li, S., He, J., Qiao, M. | Characteristics of oxytetracycline sorption and potential bioavailability in soils with various physical-chemical properties | Chemosphere | 2012 | 87 | 542–548 |
| Kristiansson, E., Fick, J., Janzon, A., Grabic, R., Rutgersson, C., Weijdegard, B., Soderstrom, H., Larsson, D.G.J. | Pyrosequencing of antibiotic-contaminated river sediments reveals high levels of resistance and gene transfer elements | Plos One | 2011 | 6 | e17038 |
| Kubota, K., Hayashi, M., Matsunaga, K., Iguchi, A., Ohashi, A., Li, Y.Y., Yamaguchi, T., Harada, H. | Microbial community composition of a down-flow hanging sponge (DHS) reactor combined with an up-flow anaerobic sludge blanket (UASB) reactor for the treatment of municipal sewage | Bioresource Technol. | 2014 | 151 | 144–150 |
| Lee, S.H., Park, J.H., Kang, H.J., Lee, Y.H., Lee, T.J., Park, H.D. | Distribution and abundance of Spirochaetes in full-scale anaerobic digesters | Bioresource Technol. | 2013 | 145 | 25–32 |
| Lee, S.H., Park, J.H., Kim, S.H., Yu, B.J., Yoon, J.J., Park, H.D. | Evidence of syntrophic acetate oxidation by Spirochaetes during anaerobic methane production | Bioresource Technol. | 2015 | 190 | 543–549 |
| Lee, Y.Y., Lee, J.K., Park, K.H., Kim, S.Y., Roh, S.W., Lee, S.B., Choi, Y., Lee, S.J. | Paenalcaligenes hermetiae sp nov., isolated from the larval gut of Hermetia illucens (Diptera: Stratiomyidae), and emended description of the genus Paenalcaligenes | Int. J. Syst. Evol. Micr. | 2013 | 63 | 4224–4229 |
| Li, B., Yang, Y., Ma, L.P., Ju, F., Guo, F., Tiedje, J.M., Zhang, T. | Metagenomic and network analysis reveal wide distribution and co-occurrence of environmental antibiotic resistance genes | ISME J. | 2015 | 9 | 2490–2502 |
| Li, D.H., Luo, R.B., Liu, C.M., Leung, C.M., Ting, H.F., Sadakane, K., Yamashita, H., Lam, T.W. | MEGAHIT v1.0: A fast and scalable metagenome assembler driven by advanced methodologies and community practices | Methods | 2016 | 102 | 3–11 |
| Li, J., Rui, J., Yao, M., Zhang, S., Yan, X., Wang, Y., Yan, Z., Li, X. | Substrate type and free ammonia determine bacterial community structure in full-scale mesophilic anaerobic digesters treating cattle or swine manure | Front. Microbiol. | 2015 | 6 | 1337 |
| Li, M.J., Wang, Y.H., Liu, Z.W., Sha, Y., Korshin, G.V., Chen, Y.C. | Metal-release potential from iron corrosion scales under stagnant and active flow, and varying water quality conditions | Water Res. | 2020 | 175 | 115675 |
| Liao, H.P., Lu, X.M., Rensing, C., Friman, V.P., Geisen, S., Chen, Z., Yu, Z., Wei, Z., Zhou, S.G., Zhu, Y.G. | Hyperthermophilic composting accelerates the removal of antibiotic resistance genes and mobile genetic elements in sewage sludge | Environ. Sci. Technol. | 2018 | 52 | 266–276 |
| Liu, Z., Klumper, U., Liu, Y., Yang, Y., Wei, Q., Lin, J.G., Gu, J.D., Li, M. | Metagenomic and metatranscriptomic analyses reveal activity and hosts of antibiotic resistance genes in activated sludge | Environ. Int. | 2019 | 129 | 208–220 |
| Lu, C., Gu, J., Wang, X., Liu, J., Zhang, K., Zhang, X., Zhang, R. | Effects of coal gasification slag on antibiotic resistance genes and the bacterial community during swine manure composting | Bioresoure Technol. | 2018 | 268 | 20–27 |
| Luo, G., Li, B., Lo, L.G., Zhang, T., Angelidaki, I. | Antibiotic resistance genes and correlations with microbial community and metal resistance genes in full-scale biogas reactors as revealed by metagenomic analysis | Environ. Sci. Technol. | 2017 | 51 | 4069–4080 |
| Ma, L.P., Li, B., Jiang, X.T., Wang, Y.L., Xia, Y., Li, A.D., Zhang, T. | Catalogue of antibiotic resistome and host-tracking in drinking water deciphered by a large scale survey | Microbiome | 2017 | 5 | 154 |
| Ma, L.P., Xia, Y., Li, B., Yang, Y., Li, L.G., Tiedje, J.M., Zhang, T. | Metagenomic assembly reveals hosts of antibiotic resistance genes and the shared resistome in pig, chicken, and human Feces | Environ. Sci. Technol. | 2016 | 50 | 420–427 |
| Ma, Y.J., Wilson, C.A., Novak, J.T., Riffat, R., Aynur, S., Murthy, S., Prudens, A. | Effect of various sludge digestion conditions on sulfonamide, macrolide, and tetracycline resistance genes and class I integrons | Environ. Sci. Technol. | 2011 | 45 | 7855–7861 |
| Marti, R., Dabert, P., Ziebal, C., Pourcher, A.M. | Evaluation of Lactobacillus sobrius/ L. amylovorus as a new microbial marker of pig manure | Appl. Environ. Microb. | 2010 | 76 | 1456–1461 |
| Masse, D., Gilbert, Y., Topp, E. | Pathogen removal in farm-scale psychrophilic anaerobic digesters processing swine manure | Bioresource Technol. | 2011 | 102 | 641–646 |
| Mata-Alvarez, J., Dosta, J., Romero-Güiza, M.S., Fonoll, X., Peces, M., Astals, S. | A critical review on anaerobic co-digestion achievements between 2010 and 2013 | Renew. Sust. Energy Rev. | 2014 | 36 | 412–427 |
| Moura, A., Soares, M., Pereira, C., Leitao, N., Henriques, I., Correia, A. | INTEGRALL: a database and search engine for integrons, integrases and gene cassettes | Bioinformatics | 2009 | 25 | 1096–1098 |
| Munck, C., Albertsen, M., Telke, A., Ellabaan, M., Nielsen, P.H., Sommer, M.O.A. | Limited dissemination of the wastewater treatment plant core resistome | Nat. Commun. | 2015 | 6 | 8452 |
| Nelson, M.C., Morrison, M., Yu, Z.T. | A meta-analysis of the microbial diversity observed in anaerobic digesters | Bioresource Technol. | 2011 | 102 | 3730–3739 |
| Pal, C., Bengtsson-Palme, J., Rensing, C., Kristiansson, E., Larsson, D.G.J. | BacMet: antibacterial biocide and metal resistance genes database | Nucleic Acids Res. | 2014 | 42 | D737–D743 |
| Parks, D.H., Imelfort, M., Skennerton, C.T., Hugenholtz, P., Tyson, G.W. | CheckM: assessing the quality of microbial genomes recovered from isolates, single cells, and metagenomes | Genome Res. | 2015 | 25 | 1043–1055 |
| Qian, X., Wang, Z., Shen, G., Chen, X., Tang, Z., Guo, C., Gu, H., Fu, K. | Heavy metals accumulation in soil after 4 years of continuous land application of swine manure: A field-scale monitoring and modeling estimation | Chemosphere | 2018 | 210 | 1029–1034 |
| Resende, J.A., Diniz, C.G., Silva, V.L., Otenio, M.H., Bonnafous, A., Arcuri, P.B., Godon, J.J. | Dynamics of antibiotic resistance genes and presence of putative pathogens during ambient temperature anaerobic digestion | J. Appl. Microbiol. | 2014 | 117 | 1689–1699 |
| Sarmah, A.K., Meyer, M.T., Boxall, A.B. | A global perspective on the use, sales, exposure pathways, occurrence, fate and effects of veterinary antibiotics (VAs) in the environment | Chemosphere | 2006 | 65 | 725–759 |
| Shankar, C., Mathur, P., Venkatesan, M., Pragasam, A.K., Anandan, S., Khurana, S., Veeraraghavan, B. | Rapidly disseminating blaOXA-232 carrying Klebsiella pneumoniae belonging to ST231 in India: multiple and varied mobile genetic elements | BMC Microbiol. | 2019 | 19 | 137 |
| Shi, P., Jia, S.Y., Zhang, X.X., Zhang, T., Cheng, S.P., Li, A.M. | Metagenomic insights into chlorination effects on microbial antibiotic resistance in drinking water | Water Res. | 2013 | 47 | 111–120 |
| Siguier, P., Perochon, J., Lestrade, L., Mahillon, J., Chandler, M. | ISfinder: the reference centre for bacterial insertion sequences | Nucleic Acids Res. | 2006 | 34 | D32–D36 |
| Silva, F., Queiroz, J.A., Domingues, F.C. | Evaluating metabolic stress and plasmid stability in plasmid DNA production by Escherichia coli | Biotechnol. Adv. | 2012 | 30 | 691–708 |
| Su, J.Q., An, X.L., Li, B., Chen, Q.L., Gillings, M.R., Chen, H., Zhang, T., Zhu, Y.G. | Metagenomics of urban sewage identifies an extensively shared antibiotic resistome in China | Microbiome | 2017 | 5 | 84 |
| Sun, W., Gu, J., Wang, X.J., Qian, X., Tuo, X.X. | Impacts of biochar on the environmental risk of antibiotic resistance genes and mobile genetic elements during anaerobic digestion of cattle farm wastewater | Bioresource Technol. | 2018 | 256 | 342–349 |
| Sundberg, C., Al-Soud, W.A., Larsson, M., Alm, E., Yekta, S.S., Svensson, B.H., Sorensen, S.J., Karlsson, A. | 454 pyrosequencing analyses of bacterial and archaeal richness in 21 full-scale biogas digesters | FEMS Microbiol. Ecol. | 2013 | 85 | 612–626 |
| Tian, Z., Zhang, Y., Yu, B., Yang, M. | Changes of resistome, mobilome and potential hosts of antibiotic resistance genes during the transformation of anaerobic digestion from mesophilic to thermophilic | Water Res. | 2016 | 98 | 261–269 |
| Urban, M., Cuzick, A., Rutherford, K., Irvine, A., Pedro, H., Pant, R., Sadanadan, V., Khamari, L., Billal, S., Mohanty, S., Hammond-Kosack, K.E. | PHI-base: a new interface and further additions for the multi-species pathogen-host interactions database | Nucleic Acids Res. | 2017 | 45 | D604–D610 |
| Veeresh, G.S., Kumar, P., Mehrotra, I. | Treatment of phenol and cresols in upflow anaerobic sludge blanket (UASB) process: a review | Water Res. | 2005 | 39 | 154–170 |
| Wallace, J.S., Garner, E., Pruden, A., Aga, D.S. | Occurrence and transformation of veterinary antibiotics and antibiotic resistance genes in dairy manure treated by advanced anaerobic digestion and conventional treatment methods | Environ. Pollut. | 2018 | 236 | 764–772 |
| Wang, R., Chen, M.X., Feng, F., Zhang, J.Y., Sui, Q.W., Tong, J., Wei, Y.S., Wei, D.B. | Effects of chlortetracycline and copper on tetracyclines and copper resistance genes and microbial community during swine manure anaerobic digestion | Bioresource Technol. | 2017 | 238 | 57–69 |
| Wang, T., Jiang, X., Feng, C., Li, A., Dong, H., Wu, S., Zheng, B. | Whole genome sequencing uncovers a novel IND-16 metallo-β-lactamase from an extensively drug-resistant Chryseobacterium indologenes strain J31 | Gut Pathog. | 2016 | 8 | 47 |
| Yang, Y., Jiang, X.T., Chai, B.L., Ma, L.P., Li, B., Zhang, A.N., Cole, J.R., Tiedje, J.M., Zhang, T. | ARGs-OAP: online analysis pipeline for antibiotic resistance genes detection from metagenomic data using an integrated structured ARG-database | Bioinformatics | 2016 | 32 | 2346–2351 |
| Zhang, J.Y., Liu, J.B., Lu, T.D., Shen, P.H., Zhong, H., Tong, J., Wei, Y.S. | Fate of antibiotic resistance genes during anaerobic digestion of sewage sludge: Role of solids retention times in different configurations | Bioresource Technol. | 2019 | 274 | 488–495 |
| Zhang, M., He, L.Y., Liu, Y.S., Zhao, J.L., Liu, W.R., Zhang, J.N., Chen, J., He, L.K., Zhang, Q.Q., Ying, G.G. | Fate of veterinary antibiotics during animal manure composting | Sci. Total Environ. | 2019 | 650 (Part 1) | 1363–1370 | Environment International 155 (2021) 106694

| Author(s) | Title | Journal | Volume | Page |
|-----------|-------|---------|--------|------|
| Zhang, M., He, L.Y., Liu, Y.S., Zhao, J.L., Zhang, J.N., Chen, J., Zhang, Q.Q., Ying, G.G. | Variation of antibiotic resistome during commercial livestock manure composting | Environ. Int. | 136 | 105458 |
| Zhang, M., Liu, Y.S., Zhao, J.L., Liu, W.R., He, L.Y., Zhang, J.N., Chen, J., He, L.K., Zhang, Q.Q., Ying, G.G. | Occurrence, fate and mass loadings of antibiotics in two swine wastewater treatment systems | Sci. Total Environ. | 639 | 1421–1431 |
| Zhang, Q.Q., Ying, G.G., Pan, C.G., Liu, Y.S., Zhao, J.L. | Comprehensive evaluation of antibiotics emission and fate in the river basins of China: Source analysis, multimedia modeling, and linkage to bacterial resistance | Environ. Sci. Technol. | 49 | 6772–6782 |
| Zhang, T., Yang, Y., Pruden, A. | Effect of temperature on removal of antibiotic resistance genes by anaerobic digestion of activated sludge revealed by metagenomic approach | Appl. Microbiol. Biotechnol. | 99 | 7771–7779 |
| Zhao, R.X., Feng, J., Yin, X.L., Liu, J., Fu, W.J., Berendonk, T.U., Zhang, T., Li, X.Y., Li, B. | Antibiotic resistome in landfill leachate from different cities of China deciphered by metagenomic analysis | Water Res. | 134 | 126–139 |
| Zhou, L.J., Ying, G.G., Liu, S., Zhao, J.L., Chen, F., Zhang, R.Q., Peng, F.Q., Zhang, Q.Q. | Simultaneous determination of human and veterinary antibiotics in various environmental matrices by rapid resolution liquid chromatography-electrospray ionization tandem mass spectrometry | J. Chromatogr. A | 1244 | 123–138 |
| Zhou, L.J., Ying, G.G., Zhang, R.Q., Liu, S., Lai, H.J., Chen, Z.F., Yang, B., Zhao, J.L. | Use patterns, excretion masses and contamination profiles of antibiotics in a typical swine farm, south China | Environ. Sci. Processes Impacts. | 15 | 802–813 | 